Tasks/AzureRmWebAppDeployment/LegacyUtils/VariableSubstituter-Legacy.ps1

```diff
@@ -0,0 +1,84 @@
+﻿function Substitute-Variables
+{  
+
+    Param(
+        [string][Parameter(mandatory=$true)] 
+        $WebAppFolderPath,
+        [string][Parameter(mandatory=$true)] 
+        $ConfigFileRegex
+    )
+
+    # Load modules
+    . $PSScriptRoot/XMLSubstituter-Legacy.ps1 -Force
+
+    # Defined supported MIME type
+    $ApplicationXmlMimeType = "application/xml"
+    $TextXmlMimeType = "text/xml"
+    $ConfigExtension = ".config"
+
+    # Find all configuration files
+    $webConfigFiles = Get-ChildItem -Path $WebAppFolderPath -Filter $ConfigFileRegex -Recurse -ErrorAction SilentlyContinue -Force
```

Comment:
The xml substitution logic seem to find files using our lib helper and not trying to find all these Mime types. Why it has to be two different logic for variable substitution and xml transformation?